gatt_assigned_characteristics.txt

```diff
@@ -0,0 +1,217 @@
+# This file holds a list of human-readable names for standardized GATT
```

Comment:
Here and in the other files can you reference the revision of the Bluetooth Assigned Numbers specification these values are based on?

Reply:
These values are not based on the Bluetooth Assigned Numbers document. They're based on what is currently in Chromium to better reflect what is happening in the real world for now. It was updated last in June 2018. See https://chromium-review.googlesource.com/c/chromium/src/+/1100126/

Note that the Bluetooth Assigned Numbers document latest revision date is `2021-08-25`
As noted earlier, we still need to create a policy to update/remove names.

Reply:
That makes sense to me. Can you note that in these files so it is clear where this data came from?

Reply:
Done in https://github.com/WebBluetoothCG/registries/pull/29/commits/54f04d4b1c25c8394f171418193eeeec4cc35d80

Reply:
If that looks good to you, I'll merge this PR and https://github.com/WebBluetoothCG/web-bluetooth/pull/560

Reply:
I think this text can cite the previous URL https://www.bluetooth.org/en-us/specification/assigned-numbers which was the source of the numbers added to chromium.  The current wording states they are not based on the assigned number document.... but they were at the time they were added to chromium.  I think that should be made clear.